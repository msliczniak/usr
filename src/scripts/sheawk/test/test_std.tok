k=foo

/usr/bin/awk
\

'BEGIN
{

v
=
'"\"$k\""'\

",bar,baz"

}

'\

'/^X
\/
\'\''X$/
{
print
}

END
{

print
"'\''"

print
"\'\''"

print
"\""

print
n

print
v



print
"?\?-"

print
"??-"


}

'\

n=xyz

