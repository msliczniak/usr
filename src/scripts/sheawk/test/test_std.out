X  /	'X
'
'
"
xyz
foo,bar,baz
??-
??-
